modules/test/playwright/tests/object-web/objectFolder.spec.ts

```diff
@@ -353,8 +347,11 @@ test.describe('manage object definitions through view object definitions', () =>
 
 		await viewObjectDefinitionsPage.deleteObjectFolder(objectFolder.name);
 
-		createdEntities.objectFolders.splice(
-			createdEntities.objectFolders.indexOf(objectFolder)
+		apiHelpers.data.splice(
+			apiHelpers.data.findIndex(
+				(object) => object.id === objectFolder.id
```

Comment:
Could we use something like:

```
apiHelpers.data.findIndex({id: objectFolder.id, type: 'objectFolder'})
```

?

Reply:
I think that we can't do something like that on `findIndex`